include/tvm/relax/utils.h

```diff
@@ -110,6 +110,11 @@ TVM_DLL Function CopyWithNewVars(Function func);
  */
 Expr ToNonDataflow(const Expr& e);
 
+/*!
+ * \brief Get the value bound in the binding.
+ */
+Expr GetBoundValue(const Binding& b);
```

Comment:
It's since been merged in another change, so I guess it's moot for now. FWIW I think hoisting the value field is the right choice, but I tried it and it breaks _a lot_ of uses. Probably still worth doing.

Reply:
If you'd like, https://github.com/apache/tvm/pull/15992 has an implementation of hoisting the `value` field to `BindingNode`, which was [passing all unit tests](https://ci.tlcpack.ai/blue/organizations/jenkins/tvm-unity/detail/PR-15992/1/pipeline/).  I closed it after feedback, but it could be worth revisiting.

Reply:
Interesting objection. I don't think there's a pressing need to change it with the utility method, though it might make sense to do it.